doc/classes/InputEvent.xml

```diff
@@ -30,14 +31,14 @@
 			<return type="int">
 			</return>
 			<description>
-				Returns the id of the device that generated the event.
+				Returns the device's id that generated the event.
```

Comment:
This is misleading - the _ID_ doesn't generate the event, the _device_ does.

Also, note get/set descriptions will be replaced by the member var's description.